homeassistant/components/plugwise/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "plugwise",
   "name": "Plugwise",
   "documentation": "https://www.home-assistant.io/integrations/plugwise",
-  "requirements": ["plugwise==0.8.5"],
+  "requirements": ["plugwise==0.16.1"],
```

Comment:
FYI, plugwise v0.16.2 was released a few hours ago.

Reply:
Not according GitHub

<img width="725" alt="CleanShot 2022-02-06 at 19 59 53@2x" src="https://user-images.githubusercontent.com/195327/152697005-226a2e21-60e3-47fd-9c00-b245a95a124b.png">

Reply:
It's on Pypi, I will add the release-info soon.

Reply:
Now it's there.

Reply:
Thanks!

Reply:
Bumped!